Framework/Algorithms/src/Rebin.cpp

```diff
@@ -135,15 +136,25 @@ std::map<std::string, std::string> Rebin::validateInputs() {
   // validate the rebin params, and outside default mode, reset them
   MatrixWorkspace_sptr inputWS = getProperty(PropertyNames::INPUT_WKSP);
   std::vector<double> rbParams = getProperty(PropertyNames::PARAMS);
-  try {
-    std::vector<double> validParams = rebinParamsFromInput(rbParams, *inputWS, g_log, binMode);
-    // if the binmode has been set, force the rebin params to be consistent
-    if (binMode != BinningMode::DEFAULT) {
-      setProperty(PropertyNames::PARAMS, validParams);
-      rbParams = validParams;
+  if (inputWS == nullptr) {
+    // The workspace could exist, but not be a MatrixWorkspace, e.g. it might be
+    // a group workspace. In that case we don't want a validation error for the
+    // Rebin parameters.
+    std::string inputWsName = getProperty(PropertyNames::INPUT_WKSP);
+    const bool wsExists = AnalysisDataService::Instance().doesExist(inputWsName);
+    if (!wsExists)
+      helpMessages[PropertyNames::INPUT_WKSP] = "Input workspace not in ADS.";
```

Comment:
I think this is actually clearer if this is one line. It makes clear that we're checking the workspace exists explicitly in the ADS.
```suggestion
    if (!AnalysisDataService::Instance().doesExist(inputWsName)) {
      helpMessages[PropertyNames::INPUT_WKSP] = "Input workspace not in ADS.";
    }
```
As an aside, I've also suggested that curly braces be added in. It's not in our standards but I think it should be. Single line if's without them look initially neater but can easily lead to problems down the line if more statements get added without due care. David Fairbrother sent me an article about a big security issue caused by these that happened to Apple a while back. Worth a read if you get time. I found it interesting, at least: https://www.imperialviolet.org/2014/02/22/applebug.html